### PR TITLE
Add How to tag guide

### PR DESCRIPTION
**Pull request by kevin27luedemann**
We added the guide on how to TAG using the skill tree. This is a vital piece for working with the skill tree.
This guide will be updated in the future, but this is a fully functioning guide.
Future updates will include more examples.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,3 +70,73 @@ The newest version of freeplane (1.11.10) is needed.
 **There are no learning objectives or tests for an overview branch. The overview branch lets trainers pick and choose relevant topics within a branch to teach at any level required so it doesn’t need to complete the learning objectives that are strictly specified in the leaf sections.** 
 
 **In Overview, only the Header and Description are required.**
+
+## How to TAG using the skill tree
+
+Before showing examples, we are reiterating important rules and definitions of skills and the skill tree.
```
Comment (cmeesters) [inline README.md:76]: Wouldn't it be better to have _one_ browsable document?

File: README.md [hunk 2]
```diff
@@ -70,3 +70,73 @@ The newest version of freeplane (1.11.10) is needed.
...
+All the rules can be found in the skill tree [Readme.md](https://github.com/HPC-certification-forum/skill-tree/blob/master/README.md).
+
+### Skills
+
+A skill is a learnable unit of content, which can be taught in no more then 4 hours and no less then 1 hours.
+A skill does not have a difficulty but can have requirements.
```
Comment (cmeesters) [inline README.md]: It does not have a difficulty ... 

Instaead of repeating "A skill"

File: README.md [hunk 3]
```diff
@@ -70,3 +70,73 @@ The newest version of freeplane (1.11.10) is needed.
...
+
+If the content of a skill has more content then or grows beyond 4 hours of teaching, it will be split into sub skills.
```
Comment (cmeesters) [inline README.md]: This reads weird, how about:

If content associated with a "skill"' beyond app. 4 hours of lecture, it ought to be split into subskills.

File: README.md [hunk 4]
```diff
@@ -70,3 +70,73 @@ The newest version of freeplane (1.11.10) is needed.
...
+The simple, original name remains and becomes an overview.
```
Comment (cmeesters) [inline README.md]: The simple, original name remains and becomes an overview item.

or "... overview entry in the skill tree."?

File: README.md [hunk 5]
```diff
@@ -70,3 +70,73 @@ The newest version of freeplane (1.11.10) is needed.
...
+A network representation would be required for this to work properly.
+
+### Skill Markdown files
+
+Each skill and skill overview has a Markdown file containing information about the content as well as requirements.
+They are directly linked in the Mind Map.
+
+### Changes to the skill tree
+
+Usually, only the leaves will change, meaning they will split into smaller and smaller leaves.
```
Comment (cmeesters) [inline README.md:111]: Should we have a sentence like: "Adding new branches will require a discussion with the panel / board"? Or something like that?

File: README.md (hunk 1 above)
Comment (kevin27luedemann) [reply README.md:76]: Yes, you are right. We should put all of this information into the Wiki and link it here. We will do this after the merge, when we also update all the Markdown files for the wiki in one big leap.

File: README.md (hunk 2 above)
Comment (kevin27luedemann) [reply README.md]: Good Idea, Is updated

File: README.md (hunk 3 above)
Comment (kevin27luedemann) [reply README.md]: Yap, I agree

File: README.md (hunk 4 above)
Comment (kevin27luedemann) [reply README.md]: Yep, made it:
overview leaf in the skill tree

File: README.md (hunk 5 above)
Comment (kevin27luedemann) [reply README.md:111]: That is a good Idea. I added:
Adding new branches will require a discussion with the board, which meets regularly.

**Comment by kevin27luedemann** [comment]
We have updated the guide according to the comments made.
This can be merged now, and we will put it into the wiki later

**Comment by cmeesters** [comment]
Should I proceed, or would you rather have a review from someone more "in the loop"?

**Comment by JulianKunkel** [comment]
I think this is great!

**Comment by JulianKunkel** [comment]
so we have two people endorsing it, and as it is non-crucial, a merge.

**Pull request merged**